policyengine_us/parameters/gov/states/ma/dta/tcap/eaedc/deductions/dependent_care/age_threshold.yaml

```diff
@@ -0,0 +1,12 @@
+description: Massachusetts limits the Emergency Aid to the Elderly, Disabled and Children program to households with at least one person above this age threshold. 
```

Comment:
This defines a dependent right? 

Can we re-frame this dependent age threshold and add a parameter for the 19 year olds for are stundents